README.md

```diff
@@ -426,15 +426,16 @@ The full YAML configuration looks like:
 ```yaml
 okta:
   client:
-    connectionTimeout: 30 # seconds
-    orgUrl: "https://{yourOktaDomain}" # i.e. https://dev-123456.okta.com
+    connectionTimeout: 30               # seconds
+    jwtExpiryTime: 3600                 # seconds
```

Comment:
Remove this value from this section, it only belongs in the "OAuth 2.0" section below